### ISSUE TITLE
0crat reassigned CR after PR was merged

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#487]
See [here](https://github.com/yegor256/jpeek/pull/108). The PR was merged, 0crat paid me 15min + 15 points rating, then took the job out of scope -- fine until here. Then:

1) 0crat assigned me again, budget **15min**
2) 0crat then said that I resigned
3) 0crat said the order was successfuly finished and awarded me **30 points**. However, no payment has been repeated, just rating points added.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by amihaiemil** [comment]
@yegor256 have a look, 0crat seems to have gone haywire :) Now it has assigned that job again, to someone else.

**Comment by 0crat** [comment]
Job [#487](https://github.com/zerocracy/farm/issues/487) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/472033711). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@yegor256 could you please post footprint claims here for this job: https://github.com/yegor256/jpeek/pull/108

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-368403781))

@g4s8 The user @g4s8[/z](https://www.0crat.com/u/g4s8) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+4600](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @caarlos0[/z](https://www.0crat.com/u/caarlos0), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/662041318). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@caarlos0[/z](https://www.0crat.com/u/caarlos0) this job was assigned to you 23days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@caarlos0[/z](https://www.0crat.com/u/caarlos0) this job was assigned to you 23days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-394094509))

@g4s8 The user @caarlos0[/z](https://www.0crat.com/u/caarlos0) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1550839064); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@t-izbassar[/z](https://www.0crat.com/u/t-izbassar) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by t-izbassar** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-399547662))

@t-izbassar The user @t-izbassar[/z](https://www.0crat.com/u/t-izbassar) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar)

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1751124108); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by krzyk** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-399749113))

@krzyk The user @krzyk[/z](https://www.0crat.com/u/krzyk) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1801351096); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by amihaiemil** [comment]
@paulodamaso My bet is that this bug is not present anymore. Write a test for this case and see if you can find it or not. I would look in the stakeholder which manages the merging of a PR and try to write a testcase for that.

**Comment by paulodamaso** [comment]
@amihaiemil Thanks for the lead. @g4s8 could you please post footprint claims here for this job: yegor256/jpeek#108

**Comment by g4s8** [comment]
@paulodamaso sure: [footprint.txt](https://github.com/zerocracy/farm/files/2151536/footprint.txt)

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-403554764))

@paulodamaso The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1910830203); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@amihaiemil yes, it's the same error as in this bug: https://github.com/zerocracy/farm/issues/1396#issuecomment-403705200

**Comment by amihaiemil** [comment]
@0crat refuse

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2010865517); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by krzyk** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-406607318))

@krzyk The user @krzyk[/z](https://www.0crat.com/u/krzyk) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by carlosmiranda** [comment]
@0crat refuse

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2b3deec7-bf3d-405f-991a-1f7ef51e1965, thread: pool-399-thread-1
com.zerocracy.pm.ClaimsItem[104] java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 6 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080358403">
   <created>2018-07-27T03:52:54.856Z</created>
   <type>Cancel order</type>
   <token>github;zerocracy/farm;487;408303794</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408303794</param>
   </params>
</claim>

```

[0.27.3](https://github.com/zerocracy/farm): Issue: zerocracy/farm#487, Comment: 408303794

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2b3deec7-bf3d-405f-991a-1f7ef51e1965, thread: pool-399-thread-1
com.zerocracy.radars.github.ReSafe[81] java.io.IOException: java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 6 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080358403">
   <created>2018-07-27T03:52:54.856Z</created>
   <type>Cancel order</type>
   <token>github;zerocracy/farm;487;408303794</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408303794</param>
   </params>
</claim>

com.zerocracy.pm.ClaimsItem[104] java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 6 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080358403">
   <created>2018-07-27T03:52:54.856Z</created>
   <type>Cancel order</type>
   <token>github;zerocracy/farm;487;408303794</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408303794</param>
   </params>
</claim>

```

[0.27.3](https://github.com/zerocracy/farm): Issue: zerocracy/farm#487, Action: `created`

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2b3deec7-bf3d-405f-991a-1f7ef51e1965, thread: pool-414-thread-1
com.zerocracy.pm.ClaimsItem[104] java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 11 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Check job status;{comment=408306144, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408306144,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080419433">
   <created>2018-07-27T04:13:13.944Z</created>
   <type>Check job status</type>
   <token>github;zerocracy/farm;487;408306144</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408306144</param>
   </params>
</claim>

```

[0.27.3](https://github.com/zerocracy/farm): Issue: zerocracy/farm#487, Comment: 408306144

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@2b3deec7-bf3d-405f-991a-1f7ef51e1965, thread: pool-414-thread-1
com.zerocracy.radars.github.ReSafe[81] java.io.IOException: java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 11 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Check job status;{comment=408306144, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408306144,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080419433">
   <created>2018-07-27T04:13:13.944Z</created>
   <type>Check job status</type>
   <token>github;zerocracy/farm;487;408306144</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408306144</param>
   </params>
</claim>

com.zerocracy.pm.ClaimsItem[104] java.lang.IllegalStateException: Duplicate claims are not allowed in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), can't add this XML to 11 existing ones (Cancel order;{comment=408303794, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408303794,Check job status;{comment=408306144, job=gh:zerocracy/farm#487, issue=487, repo=zerocracy/farm};github;zerocracy/farm;487;408306144,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};slack;C3T49A35L;UAQRSJDAM;direct,Notify in Slack;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349317, job=gh:zerocracy/farm#1488, login=krzyk, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to @krzyk[/z](https://www.0crat.com/u/krzyk)[/z](https://www.0crat.com/u/krzyk) (role is `REV`), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307), flow=Ping; Start order; Order was given; Notify project};slack;C3NDPUA8L,Notify in Telegram;{reason=2080347307, role=REV, public=true, estimate=$4.00, cause=2080349316, job=gh:zerocracy/farm#1488, message=The job [#1488](https://github.com/zerocracy/farm/issues/1488) was assigned to you in [Zerocracy](https://www.0crat.com/p/C3NDPUA8L) as `REV` a minute ago; here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2080347307);you will earn $4.00 on completion, login=krzyk, flow=Ping; Start order; Order was given; Notify user};telegram;553846308,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};,Ping;{};):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="2080419433">
   <created>2018-07-27T04:13:13.944Z</created>
   <type>Check job status</type>
   <token>github;zerocracy/farm;487;408306144</token>
   <author>carlosmiranda</author>
   <params>
      <param name="job">gh:zerocracy/farm#487</param>
      <param name="repo">zerocracy/farm</param>
      <param name="issue">487</param>
      <param name="comment">408306144</param>
   </params>
</claim>

```

[0.27.3](https://github.com/zerocracy/farm): Issue: zerocracy/farm#487, Action: `created`

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-408303794))

@carlosmiranda The user @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) resigned from [#487](https://github.com/zerocracy/farm/issues/487), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-408306144))

@carlosmiranda This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#487](https://github.com/zerocracy/farm/issues/487) is [in scope](http://www.zerocracy.com/policy.html#14) for 6mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @paulodamaso[/z](https://www.0crat.com/u/paulodamaso): User was resigned from the ticket
    * @amihaiemil[/z](https://www.0crat.com/u/amihaiemil): User was resigned from the ticket
    * @krzyk[/z](https://www.0crat.com/u/krzyk): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23487%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) assigned to @rgevaerd[/z](https://www.0crat.com/u/rgevaerd), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2091722387); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by rgevaerd** [comment]
@0crat waiting on PR #1531

**Comment by 0crat** [comment]
@rgevaerd[/z](https://www.0crat.com/u/rgevaerd) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by rgevaerd** [comment]
@0crat waiting on PR #1531

**Comment by 0crat** [comment]
> @0crat waiting on PR #1531 ([here](https://github.com/zerocracy/farm/issues/487#issuecomment-410119255))

@rgevaerd The impediment for [#487](https://github.com/zerocracy/farm/issues/487) was registered successfully by @rgevaerd[/z](https://www.0crat.com/u/rgevaerd)

**Comment by rgevaerd** [comment]
@amihaiemil  #1531 has been merged, can we close this now?

**Comment by 0crat** [comment]
The job [#487](https://github.com/zerocracy/farm/issues/487) is now out of scope

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @rgevaerd[/z](https://www.0crat.com/u/rgevaerd), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @rgevaerd[/z](https://www.0crat.com/u/rgevaerd)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
